### ISSUE TITLE
Please Add LookingGlass Functionality

### ISSUE THREAD

**Issue by bauerbach** [Geforce132/SecurityCraft#69]
LookingGlass is no longer supported apparently and will never be updated to 1.12.2,, so can you please implement the LookingGlass features into the mod so that we can use the Frames in 1.12.2?

**Comment by Eonasdan** [comment]
There is a 1.12.2 branch on looking glass, however I couldn't get it to work properly

**Comment by darthvader45** [comment]
I'm interested in this as well.  Can't stand not being able to put monitors in frames.

**Comment by Scwestbrook** [comment]
Same here, but I'm on 1.16.4

**Comment by Geforce132** [comment]
Just giving an update about this request here. We all want "real" camera monitors back so that everyone can set up those cool security rooms again. Unfortunately, xCompWiz recently mentioned that LookingGlass will most likely not be worked on even with his revamped version of Mystcraft coming soon. So... getting an updated port of LookingGlass anytime soon is probably not going to happen. We may look into using a LookingGlass-style alternative such as [GLASS](https://www.curseforge.com/minecraft/mc-mods/general-laymans-aesthetic-spying-screen-glass) in the future (if any of you have suggestions as to libraries to use, please let us know!), but this feature is not something being actively worked on right now, unfortunately.

**Comment by Scwestbrook** [comment]
Thanks for trying, this might become one of the updates in the next five years.

**Comment by Scwestbrook** [comment]
Hi, it appears that the forge/fabric mod [Doors Of Infinity](https://www.curseforge.com/minecraft/mc-mods/doors-of-infinity) includes photon transmitters which are creating views of other dimensions. Perhaps this could be helpful? Has 1.14, 1.15, 1.16 functionality.

**Comment by bl4ckscor3** [comment]
It depends on Immersive Portals and thus is a completely different thing from what we want to achieve (think about it as looking through a door versus looking onto a screen - latter of which is our goal). Ideally, I would like to not have to depend on other libraries that may stop updating like LookingGlass did.

**Comment by darthvader45** [comment]
Yeah, I've been wondering whether or not it might be possible to integrate LookingGlass into Securitycraft, thus eliminating the need for it as a separate dependency.  Sure it'd slow development considerably, but perhaps it can be a distant future thing?

**Comment by darthvader45** [comment]
So that means your team is actually working on LookingGlass now as well as Securitycraft?  If you actually are doing that, mad props to you guys.

**Comment by bl4ckscor3** [comment]
I never said we were working on LookingGlass and I don't know how my message implied that we would be. I said that we are writing all the code that we need to make the feature (that was made possible by LookingGlass) work ourselves without relying on any 3rd party library. LookingGlass is not the only way for this to work.

**Comment by darthvader45** [comment]
Oh.  Still, mad props for working on making the feature work w/o LookingGlass. Good luck.

**Comment by Scwestbrook** [comment]
@bl4ckscor3, when I referenced the mod, I meant if immersive portals can render other dimensions, you could take a look at it's code. It's open-source (I think.)

**Comment by Xious01** [comment]
May I say that you could use immersive portals instead. Pretty much the same thing.

**Comment by Scwestbrook** [comment]
Xious01, in reply to your comment, look at Geforce123's earlier post:
> It depends on Immersive Portals and thus is a completely different thing from what we want to achieve (think about it as looking through a door versus looking onto a screen - latter of which is our goal). Ideally, I would like to not have to depend on other libraries that may stop updating like LookingGlass did.

**Comment by darthvader45** [comment]
Agreed, if LookingGlass can be built into Securitycraft going forward, it would remove the dependency while re-adding functionality that some players wish to see return.

**Comment by bl4ckscor3** [comment]
Again, we will not integrate LookingGlass into SecurityCraft. We will be writing our own code to make the feature possible.
I will lock this thread as the discussion going on here is not usefule.

**Comment by bl4ckscor3** [comment]
This will be part of the next SecurityCraft update.
